src/main/java/seedu/address/model/contact/ContactPhone.java

```diff
@@ -11,8 +11,8 @@ public class ContactPhone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be 8 digits long";
+    public static final String VALIDATION_REGEX = "\\d{8}";
```

Comment:
Will 8 digit limit the use of this application? Hypothetically speaking what if I have foreigner who is coming to Singapore request a freelancer?

Reply:
Maybe thinking about having foreign contacts will be a little far fetched? 
I think foreign counterparts hiring freelancers would be a rarity, should they work in Singapore, I believe they would have engaged a Singaporean line? 
What do u think?

Reply:
I do see it as a possibility! Personally, I have had foreign phone numbers that I WhatsApp to ask about the details! However if we are targeting mainly the SG Market then I would agree! 👍

Reply:
Perhaps a warning could be shown to the user if the phone number entered is more than 8 digits? It could avoid [overzealous input validation](https://nus-cs2103-ay2223s2.github.io/website/admin/tp-pe.html#feature-flaws-5).

Reply:
I think that's a really good suggestion made by Gloria, perhaps we can try going in that direction?

Reply:
That's a good point @glozxi, as of our current implementation, it only allows for phone numbers (originally was >3 characters, must be number). I'm concerned that implementing use cases like `1234 5678 (HP) 1111-3333 (Office)`  will affect how we define contact (given that a contact has a `name` and a `number` and the way the linkage is made.

Reply:
I was thinking that we could continue to just have numbers for phone numbers.